telegram-bot/meetup.go

```diff
@@ -90,6 +120,23 @@ func nextmeetup(ID int64, client mongo.Client) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	nxtmeetupdata := "Details of next OSDC Meetup :" + "\n" + "Title -" + "\t" + result.Name + "\n" + "Date -" + "\t" + result.Date + "\n" + "Venue -" + result.Venue
-	bot.Send(tbot.NewMessage(ID, nxtmeetupdata))
+	timeString := result.Date.Local().Format("Mon Jan _2 15:04:05 2006")
+	nxtMeetupData := "Details of next OSDC Meetup :" + "\n" + "Title -" + "\t" + result.Name + "\n" + "Date -" + "\t" + timeString + "\n" + "Venue - " + result.Venue
+	bot.Send(tbot.NewMessage(ID, nxtMeetupData))
+}
+
+func reminder(ID int64, client mongo.Client, s1 *gocron.Scheduler) {
+	collection := client.Database("test").Collection("meetups")
+	var result meetupdata
```

Comment:
What does `result` mean here?

Reply:
Borrowed few lines introduced in #20 from the `nextmeetup` function. :grin:
```suggestion
	var data meetupdata
```